modelmatrix-core/src/main/scala/com/collective/modelmatrix/ModelMatrix.scala

```diff
@@ -72,16 +72,14 @@ object ModelMatrix extends ModelMatrixUDF {
         build()
   }
 
-  trait PostgresModelMatrixCatalog extends ModelMatrixCatalogAccess {
 
-    protected val driver = slick.driver.PostgresDriver
-    import driver.api._
+  trait DbModelMatrixCatalog extends ModelMatrixCatalogAccess {
```

Comment:
I think ModelMatrixCatalogAccess inheritance can be removed, and just move implementation from DbModelMatrixCatalog inside CatalogAccess. I don't think that some other inheritance is possible.